tests/Microsoft.Bot.Builder.Core.Extensions.Tests/ActivityFilter_MiddlewareTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Schema;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Bot.Builder.Core.Extensions.Tests
+{
+    [TestClass]
+    public class ActivityFilter_MiddlewareTests
```

Comment:
Consider adding a test that registers:
1. Null activity type. Make sure an excpetion is thrown.
2. Empty activity type. Make sure an exception is thrown.

Reply:
Tests added.